client/http/client.go

```diff
@@ -196,13 +218,19 @@ func (c *client) execDuration(name string, duration time.Duration) {
 	c.executionDuration.WithLabelValues(name).Observe(duration.Seconds())
 }
 
+// Header key definition constants.
+const (
+	pdAllowFollowerHandleKey = "PD-Allow-Follower-Handle"
+	componentSignatureKey    = "component"
```

Comment:
We can have a more intuitive name like 'CallerID', and it's a filed in http header, "X-Caller-ID" maybe a better name according HTTP RFC docs @JmPotato

Reply:
Using `component` here is to reuse https://github.com/tikv/pd/blob/master/pkg/utils/requestutil/request_info.go#L52, but you're right about the naming, we could create a new filed to distinguish with it.